.ci/checker-framework.groovy

```diff
@@ -168,7 +168,7 @@ private static String getOsSpecificCmd(final String cmd) {
 private static List<CheckerFrameworkError> getErrorFromText(final List<List<String>> errorsList) {
     final List<CheckerFrameworkError> errors = new ArrayList<>()
     final Pattern errorExtractingPattern = Pattern
-        .compile('.*[\\\\/](checkstyle[\\\\/]src.*\\.java):\\[(\\d+)[^]]*][^\\[]*\\[([^]]*)](.*)')
+        .compile('.*[\\\\/](src[\\\\/].*\\?.java):\\[(\\d+)[^]]*][^\\[]*\\[([^]]*)](.*)')
```

Comment:
Why the new `?`? It wasn't there before. Isn't it in a wrong place as I thought `\\` before it was to escape the period to be a literal.

Reply:
>Why the new ?? It wasn't there before.

https://github.com/checkstyle/checkstyle/pull/12337#discussion_r1002857217

Reply:
lol, note to self to stop staying up so late.
Please remove the `?`. I think I was originally making this a question and that was to end the question.